.github/workflows/ci.yml

```diff
@@ -67,6 +76,19 @@ jobs:
     - name: Run pgx tests
       run: su postgres -c 'sh tools/build -pg${{ matrix.pgversion }} test-extension 2>&1'
 
+    - name: Build and install TimescaleDB
+      if: ${{ inputs.tsdb-commit != '' }}
+      run: |
+        apt-get install -y wget cmake gcc build-essential libkrb5-dev
+        git clone '${{ inputs.tsdb-repo }}'
+        cd timescaledb
+        git pull --all
+        git switch --detach ${{ inputs.tsdb-commit }}
+        mkdir build
+        cd build
+        cmake .. -DUSE_OPENSSL=0 -DLINTER=0 -DCMAKE_PROGRAM_PATH=/usr/lib/postgresql/${{ matrix.pgversion }}/bin
```

Comment:
Does this CMAKE_PROGRAM_PATH cause our cmake scripts to find the right pg_config and thereby cause `make install` to place the extension into the right place under /usr/lib and thereby clobbering the files from the deb?  That merits a comment.

Reply:
added a comment